documentation/src/main/asciidoc/reference/search-dsl-predicate.asciidoc

```diff
@@ -630,6 +630,57 @@ but can be <<search-dsl-predicate-common-boost,boosted>>,
 either on a per-field basis with a call to `.boost(...)` just after `.field(...)`/`.fields(...)`
 or for the whole predicate with a call to `.boost(...)` after `.matching(...)`.
 
+[[search-dsl-predicate-terms]]
+== `terms`: match a series of term
```

Comment:
Maybe "match a set of terms"? Series means "sequence" and might imply order, which we definitely don't support here.